en/docs/install-and-setup/setup/api-controller/migrating-applications-to-different-environments.md

```diff
@@ -73,16 +74,22 @@ You can export an application in the Developer Portal and download it as a zippe
         !!!note
             `apictl export-app` command has been depcrecated from the API Controller 4.0.0 onwards. Instead use `apictl export app` as shown above.
 
-    -   Response
-        ``` java
+    -   **Response**
+
+        ``` bash tab="Response Format"
+        Successfully exported Application!
```

Comment:
Why have you capitalized the word "Application"? As this word appears in the middle of a sentence the word should start with lowercase.

It will be good if you can change the response of the implementation from [1] to [2].

Check other 

[1] Successfully exported Application!
[2] Successfully exported the application.

Reply:
Since exporting an application is an operation to do with the **Application**, we want to emphasize the subject of the operation in the message that is displayed. I think we can keep this at is.